apps/studio/components/interfaces/Database/Schemas/Schemas.utils.ts

```diff
@@ -12,14 +12,14 @@ const NODE_SEP = 25
 const RANK_SEP = 50
 
 export async function getGraphDataFromTables(
-  ref: string,
-  schema: PostgresSchema,
-  tables: PostgresTable[]
+  ref?: string,
```

Comment:
i might find the answer as i check through down, but any reason why these need to be optional?